### PR TITLE
Weet model create

### PR DESCRIPTION
**Pull request by ricardoamoreno**
## Summary

- weet model added
- weet migration added

## Known Issues

None

## Trello Card

[Modelo de Weet](https://trello.com/c/dWKT6mFB)

### CODE REVIEW

**Review by wgermancutraro — DISMISSED** [review]

**Review by martinremis — APPROVED** [review]

File: app/models/weet.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        autoIncrement: true,
+        primaryKey: true,
+        field: 'id'
+      },
+      content: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        field: 'content'
+      }
+    },
```
Comment (martinremis) [inline app/models/weet.js:22]: You need to define the `user_id userId` field here, don't you?

**Review by martinremis — CHANGES_REQUESTED** [review]

File: app/models/weet.js (hunk 1 above)
Comment (ricardoamoreno) [reply app/models/weet.js:22]: I thought it was not necessary because I had already defined the association

Comment (martinremis) [reply app/models/weet.js:22]: I am not completely sure, but I think it is necessary for creations, updates, gets, etc. If you are sure it works anyway there is no need to add it

**Review by martinremis — APPROVED** [review]

File: app/models/weet.js (hunk 1 above)
Comment (ricardoamoreno) [reply app/models/weet.js:22]: to create it works, to update and get I have not tried it yet

**Pull request merged**